InvenTree/part/templates/part/part_pricing.html

```diff
@@ -9,11 +9,11 @@
 
 <table class='table table-striped table-condensed table-price-two'>
     <tr>
-        <td><b>{% trans 'Part' %}</b></td>
+        <td><strong>{% trans 'Part' %}</b></td>
```

Comment:
Should the `<strong>` be closed with `</strong>` and not `</b>`?

Reply:
@SchrodingersGat you absolutley right, seems like I forgot to push that